schemas/vscode-base.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Also what is the purpose of this file? (tried googling it, but found nothing on the first glance) :)

Reply:
It's just to enabled comments and trailing commas in project-wide vscode configuration files. Not that relevant for Sublime but I'm working on projects where developers split between vscode and sublime so those are common for me at least.